GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -35,6 +35,7 @@ if [ -z $VFS_PUBLISHDIR ]; then
 fi
 
 STAGINGDIR=$BUILDOUTPUTDIR"Staging"
+PACKAGESTAGINGDIR="${BUILDOUTPUTDIR}Packages"
```

Comment:
Should this have a `/` in it after `${BUILDOUTPUTDIR}`? --> `"${BUILDOUTPUTDIR}/Packages"`

Reply:
`${BUILDOUTPUTDIR}` already has the trailing `/`. I can remove it if it looks confusing.

Reply:
Ahh I missed that.  Related question, why does `STAGINGDIR=$BUILDOUTPUTDIR"Staging"` not use the `{ }` braces like `PACKAGESTAGINGDIR="${BUILDOUTPUTDIR}Packages"` does?

Can we make these consistent (I am not familiar with bash script and so I'm not sure which is preferable)?

Reply:
have clean it up a little bit, removed the trailing `/` in `BUILDOUTPUTDIR` and other inconsistencies.